src/encoded/static/components/browse/PublicationSearchView.js

```diff
@@ -0,0 +1,245 @@
+'use strict';
+
+import React, { useMemo } from 'react';
+import memoize from 'memoize-one';
+import _ from 'underscore';
+
+import { SearchView as CommonSearchView } from '@hms-dbmi-bgm/shared-portal-components/es/components/browse/SearchView';
+import { console, analytics, object, navigate as spcNavigate, valueTransforms } from '@hms-dbmi-bgm/shared-portal-components/es/components/util';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['console' is defined but never used. (no-unused-vars)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4551745)